curated_transformers/generation/logits.py

```diff
@@ -121,3 +122,43 @@ def _process_logits(self, logits: Tensor):
             return
 
         logits /= self.temperature
+
+
+class MaskTransform(LogitsTransform):
```

Comment:
I think it would be nice if the name also makes clear what we are masking, since we already have multiple masks. Maybe `MaskPiecesTransform`? `VocabMaskTransform`?

Reply:
Renamed to `VocabMaskTransform`.